packages/client/src/render/screenRouter.js

```diff
@@ -1,11 +1,20 @@
 import regexparam from "regexparam"
 import { routerStore } from "../state/store"
+import { getAppId } from "./getAppId"
 
-export const screenRouter = ({ screens, onScreenSelected, appRootPath }) => {
+export const screenRouter = ({ screens, onScreenSelected, window }) => {
```

Comment:
Is it necessary to pass `window` since it's global?

Reply:
its for the tests, need to pass the JSDom window :(

Reply:
### Ah nightmare. I wonder if we could assign it in the test:

```js
window = jsDomWindow

testStuff()..
```